install_pkg.pm

```diff
@@ -46,6 +32,12 @@ sub foo_to_pkg
 		'libattr-devel-suse' => 'libattr-devel',
 		'libcap-devel-suse' => 'libcap-devel',
 		'libnuma-devel-suse' => 'libnuma-devel',
+
+		'libaio-devel-fedora' => 'libaio-devel',
+		'libacl-devel-fedora' => 'libacl-devel',
+		'libattr-devel-fedora' => 'libattr-devel',
+		'libcap-devel-fedora' => 'libcap-devel',
+		'libnuma-devel-fedora' => 'numactl-devel',
```

Comment:
@wangli5665 nit: maybe sort distros alphabetically (I'm planning to add at least Alpine).

Reply:
Agreed  +1